dev/tools/controllerbuilder/pkg/codegen/common.go

```diff
@@ -39,7 +39,8 @@ var protoMessagesNotMappedToGoStruct = map[string]string{
 
 var Acronyms = []string{
 	"ID", "HTML", "URL", "URI", "HTTP", "HTTPS", "SSH",
-	"IP", "GB", "FS", "PD", "KMS", "GCE", "VTPM",
+	"IP", "GB", "FS", "PD", "KMS", "GCE", "VTPM", "SSL",
+	"CA", "CPU", "SQL", "PSC", "DNS",
```

Comment:
I'm a little bit lost. I think we have reviewed this code and discussed the potential risk about CA. Why do we have a separate PR?

Reply:
This PR is extracted out from https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3598 where I included a lot of more functionality in it.

Original comment is at https://github.com/GoogleCloudPlatform/k8s-config-connector/pull/3598#discussion_r1941378023
I personally don't have any concern about it. @jingyih any thoughts?

Reply:
I noticed a discrepancy between this and our test. I suggest using the same source of truth for acronyms. If there's any uncertainty about an acronym, we can check which field is currently failing to help our decision.

https://github.com/GoogleCloudPlatform/k8s-config-connector/blob/ad509808d96b5ff98bd12a78089019c8d8a2128b/tests/apichecks/crds_test.go#L166

Reply:
Good point. I think both lists contain special acronyms the other list doesn't have so I'm going to create a union of them.

Reply:
Done.